### ISSUE TITLE
wrap linked image files

### ISSUE THREAD

**Issue by eshellman** [gutenbergtools/ebookmaker#37]
local image files linked from an epub should be wrapped in html. Unwrapped images prevent upload to Google Play Books. We need to address accessibility issues that are exposed.

example: https://www.gutenberg.org/ebooks/996

**Comment by eshellman** [comment]
Fixed in 0.9
